import_and_export_data/formats/family_variant.rst

```diff
@@ -0,0 +1,16 @@
+Family variant data structure
+=============================
+
+The import/export file follows this data structure:
+
+- **code** (required): the family code
+- **family** : the family parent
```

Comment:
`The parent family` or even `the family to which this family variant belongs` as you whish

Reply:
Fix